examples/cert-manager-agent.yaml

```diff
@@ -1,3 +1,5 @@
+organization_id: "my-organization"
+cluster_id: "my_cluster"
```

Comment:
These are the same values used in the configuration file in the root of this repository:

https://github.com/jetstack/jetstack-secure/blob/24698deffc74b083115e5d3f6f342c8ec3f79399/agent.yaml#L2-L3